MAUI/MauiAppBasic/Views/MainView.xaml

```diff
@@ -1,52 +1,52 @@
-                Source="dotnet_bot.png"
-                SemanticProperties.Description="Cute dot net bot waving hi to you!"
-                HeightRequest="200"
-                HorizontalOptions="Center" />
-                
-            <Label 
-                Text="Hello, World!"
-                SemanticProperties.HeadingLevel="Level1"
-                FontSize="32"
-                HorizontalOptions="Center" />
-            
-            <Label 
-                Text="Welcome to .NET Multi-platform App UI"
-                SemanticProperties.HeadingLevel="Level2"
-                SemanticProperties.Description="Welcome to dot net Multi platform App U I"
-                FontSize="18"
-                HorizontalOptions="Center" />
-
-            <StackLayout Orientation="Horizontal" HorizontalOptions="Center">
-                <CheckBox x:Name="useEmbedded" VerticalOptions="Center"/>
-                <Label Text="Use Embedded" VerticalOptions="Center"/>
-            </StackLayout>
-
-            <Button 
-                x:Name="SignInButton"
-                Text="Sign in"
-                SemanticProperties.Hint="Sign In"
-                Clicked="OnSignInClicked"
-                HorizontalOptions="Center" />
-
-            <Button
-                x:Name="SignOutButton"
-                Text="Sign Out"
-                HorizontalOptions="Center"
-                Clicked="SignOutButton_Clicked"/>
-
-        </VerticalStackLayout>
-    </ScrollView>
- 
-</ContentPage>
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="MauiAppBasic.Views.MainView"
+             xmlns:viewmodel="clr-namespace:MauiAppBasic.ViewModels"
+             NavigationPage.HasBackButton="False"
+             Title="Login Screen"
+             >
+    <Shell.BackButtonBehavior>
+        <BackButtonBehavior IsVisible="False" IsEnabled="False" />
+    </Shell.BackButtonBehavior>
+
+    <ScrollView>
+        <VerticalStackLayout 
+            Spacing="25" 
+            Padding="30,0" 
+            VerticalOptions="Center">
+
+            <Image
+                Source="azure_active_directory.jpg"
+                SemanticProperties.Description="Azure Active Directory Logo"
+                HeightRequest="200"
+                HorizontalOptions="Center" />
+
+            <Label 
+                Text="Azure Active Directory"
+                SemanticProperties.HeadingLevel="Level1"
+                FontSize="32"
+                HorizontalOptions="Center" />
+
+            <Label 
+                Text="MAUI basic sample"
+                SemanticProperties.HeadingLevel="Level1"
+                FontSize="32"
+                HorizontalOptions="Center" />
+
+            <StackLayout Orientation="Horizontal" HorizontalOptions="Center">
+                <CheckBox x:Name="useEmbedded" VerticalOptions="Center"/>
```

Comment:
>useEmbedded [](http://example.com/codeflow?start=34&length=11)

whats this ?

Reply:
It is used on #44 in MAUI/MauiAppBasic/Views/MainView.xaml.cs